docs/sources/panels-visualizations/visualizations/bar-chart/index.md

```diff
@@ -24,6 +24,8 @@ Bar charts allow you to graph categorical data.
 
 {{< figure src="/static/img/docs/bar-chart-panel/barchart_small_example.png" max-width="1000px" caption="Bar chart" >}}
 
+{{< docs/play title="Bar and Pie Chart Visualizations in Grafana" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
```

Comment:
In the other PRs you've been using the name of the dashboard here, so we should be consistent in that to follow styling rules. Conversely, if you want to maintain the same naming convention, the casing would need to be different since we're not quoting the UI. In that case, the title should be: bar and pie chart visualizations in Grafana
```suggestion
{{< docs/play title="Grafana Bar Charts and Pie Chart" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
```